ecl/resource_ecl_provider_connectivity_tenant_connection_request_v2.go

```diff
@@ -0,0 +1,274 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/ecl/provider_connectivity/v2/tenant_connection_requests"
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceProviderConnectivityTenantConnectionRequestV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceProviderConnectivityTenantConnectionRequestV2Create,
+		Read:   resourceProviderConnectivityTenantConnectionRequestV2Read,
+		Update: resourceProviderConnectivityTenantConnectionRequestV2Update,
+		Delete: resourceProviderConnectivityTenantConnectionRequestV2Delete,
+		Schema: map[string]*schema.Schema{
+			"tenant_id_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"network_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"status": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"keystone_user_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tenant_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"description_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags_other": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"approval_request_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceProviderConnectivityTenantConnectionRequestV2Create(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Can you consider the case an user writes parameters which are not in create request(e.g. name_other, description_other) in initial template? I think they are ignored in the function. So state gets different from user's template.

Reply:
Let's consider that pattern. Thank you!

Reply:
There is no problem because there is no pattern where xxx_other is written in the creation request.